docs/relational-databases/indexes/columnstore-indexes-design-guidance.md

```diff
@@ -57,7 +57,7 @@ Consider using a clustered columnstore index when:
 
 Don't use a clustered columnstore index when:
 
-* The table requires varchar(max), nvarchar(max), or varbinary(max) data types. Or, design the columnstore index so that it doesn't include these columns.
+* The table requires varchar(max), nvarchar(max), or varbinary(max) data types. Or, design the columnstore index so that it doesn't include these columns (Only for SQL 2016 or older).

```

Comment:
```suggestion
* The table requires **varchar(max)**, **nvarchar(max)**, or **varbinary(max)** data types. Or, design the columnstore index so that it doesn't include these columns (**Applies to:** [!INCLUDE [sssql16-md](../../includes/sssql16-md.md)] and previous versions).
```